nevergrad/optimization/callbacks.py

```diff
@@ -0,0 +1,88 @@
+# Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved.
+#
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+
+import json
+import time
+import warnings
+import datetime
+from typing import Optional, Any, Union, List, Dict
+from pathlib import Path
+import numpy as np
+from . import base
+
+
+class OptimizationPrinter:
+    """Printer to register as callback in an optimizer, for printing
+    best point regularly.
+
+    Parameters
+    ----------
+    num_eval: int
+        max number of evaluation before performing another print
+    num_sec: float
+        max number of seconds before performing another print
+    """
+
+    def __init__(self, num_eval: int = 0, num_sec: float = 60) -> None:
+        self._num_eval = max(0, int(num_eval))
```

Comment:
Why the  max here?
In case that someone puts a negative number? If that's a case, I think that raising an error would be better than silently changing the value.